### PR TITLE
Add running indicator on cant close selector

### PR DESCRIPTION
**Pull request by vctt94**
This PR adds the running indicator selector into the can't close selector.

This way decrediton will not allowed to be closed if it is perfoming any action represented by the running indicator: Ticket auto buyer, running mixer or purchasing tickets

### CODE REVIEW

**Comment by alexlyp** [comment]
Hrm overall it's OK.  But the modal that I'm seeing when the mixer is running references the auto-buyer.  Can we do specific modal title/text depending on what's running?

File: app/components/modals/CantCloseModals/PurchasingTicketsModal.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    }
+    <div className={style.confirmToolbar}>
+      <KeyBlueButton
+        className={style.confirmConfirmButton}
+        onClick={onSubmit}>
+        {
+          <T
+            id="tickets.purchasing.confirmModal.btnConfirm"
+            m="Confirm"
```
Comment (jholdstock) [inline app/components/modals/CantCloseModals/PurchasingTicketsModal.jsx:32]: `Close Decrediton`

File: app/components/modals/CantCloseModals/PurchasingTicketsModal.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          id="tickets.purchasing.title"
+          m="Purchasing Tickets"
+        />
+      </div>
+    </div>
+    {
+      <div className={style.confirmContent}>
+        <T
+          id="tickets.purchasing.message"
+          m="Decrediton is finishing the purchasing ticket process and should wait."
```
Comment (jholdstock) [inline app/components/modals/CantCloseModals/PurchasingTicketsModal.jsx:20]: `Decrediton is still finalizing ticket purchases. Tickets may not be registered with the VSP if Decrediton is closed now, which can result in missed votes.`

**Comment by jholdstock** [comment]
Works well, just a couple of suggested changes to strings.

**Review by jholdstock — APPROVED** [review]

**Review by alexlyp — APPROVED** [review]

**Pull request merged**